lib/components/position_component.dart

```diff
@@ -25,6 +24,10 @@ abstract class PositionComponent extends BaseComponent {
   /// The position of this component on the screen (relative to the anchor).
   Vector2 position = Vector2.zero();
 
+  /// If the component has a [PositionComponent] as a parent it will be
+  /// added here
```

Comment:
`it will be set here.` Would be a more accurate description :-)

Reply:
Fixed